store/src/java/com/zimbra/cs/account/soap/SoapProvisioning.java

```diff
@@ -3224,5 +3239,46 @@ public List<HABGroupMember> getHABGroupMembers(Group group) throws ServiceExcept
     public void resetPassword(Account acct, String newPassword, boolean dryRun) throws ServiceException {
         throw ServiceException.UNSUPPORTED();
     }
-
+    
+    /**
+     *
+     * @param status reserved for future use if mail to send for different status to
+     *               be added
+     * @param to     accepts admins email address to which email has to be send
+     * @throws ServiceException if an error occurs while sending email for MDM
+     *                          notification mail
+     */
+    @Override
+    public String sendMdmEmail(String status, String timeInterval) throws ServiceException {
+        String devicesInfo = MailConstants.HTML_DEVICE_TABLE_INFO_HEADER;
+        String emailBody = "";
+        GetDeviceStatusResponse resp = invokeJaxb(new GetDeviceStatusRequest(null));
```

Comment:
we can add status in the request to fetch only quarantined devices, instead of pulling all devices and then checking their status in loop

Reply:
done.